### PR TITLE
Fix/23298/5 3/save confirmation anonymous

### PR DESCRIPTION
**Pull request by theodortruffer**
Discussion see: https://github.com/ILIAS-eLearning/ILIAS/pull/1720

@chfsx the changes are the same as for 5.4/trunk. The cherry-pick didn't work because of small differences in the code style (i guess).

### CODE REVIEW

File: resources [hunk 1]
```diff
@@ -0,0 +1 @@
+Customizing/global/plugins/Services/Repository/RepositoryObject/OpenCast/node_modules/paellaplayer/build/player/resources/
```
Comment (chfsx) [inline resources]: @theodortruffer I think this is by mistake?

**Review by chfsx — CHANGES_REQUESTED** [review]
@theodortruffer  see my comment about the line which should not be part of this PR

**Comment by theodortruffer** [comment]
@chfsx thanks, i removed it. Do you need a new PR or do you squash it anyway?

**Comment by chfsx** [comment]
@theodortruffer if it's ok for you I'll squash it. Since your sitting right behind my desk I'll ask you there :D

**Pull request merged**